### ISSUE TITLE
Segmentation fault with PyPlot.pygui(true)

### ISSUE THREAD

**Issue by sp94** [fonsp/Pluto.jl#494]
After upgrading to Pluto v0.11.14 I am getting a segmentation fault when using PyPlot.

Pluto input:
```julia
begin
    using PyPlot
    plot(0, 0, "o")
    gcf()
end  # causes segmentation fault
```

Console output:
```julia
julia> import Pluto; Pluto.run(1234)
Opening http://localhost:1234/ in your default browser... ~ have fun!

Press Ctrl+C in this terminal to stop Pluto

      From worker 2:	
      From worker 2:	signal (11): Segmentation fault: 11
      From worker 2:	in expression starting at none:1
      From worker 2:	_ZNK7QScreen6handleEv at /Users/samuel/.julia/conda/3/lib/libQt5Gui.5.9.6.dylib (unknown line)
Worker 2 terminated.      From worker 2:	Allocations: 13784558 (Pool: 13780097; Big: 4461); GC: 15Distributed

.ProcessExitedException(2)
```

I noticed it is doing something with `libQt5Gui` so I tried setting `pygui(false)` and that seems to fix the error
```julia
begin
    using PyPlot
    pygui(false)
    plot(0, 0, "o")
    gcf()
end  # works
```

This error occurs on
- MacOS Catalina v10.15
- Julia v1.5
- Pluto v0.11.14
- PyPlot v2.9.0

It isn't necessary for me to use `pygui(false)` on Pluto v0.10 on my Mac, or any Pluto version on my Windows 10 computer.

**Comment by fonsp** [comment]
This is more likely because of different PyPlot versions, or its dependencies, or because of different OSses. Can you try to rule that out?

**Comment by sp94** [comment]
I am using the same version of PyPlot (2.9.0) and matplotlib (3.3.1) on my Windows/Mac and the error is only occuring on my Mac.

The segfault seems to happen when using a GUI with PyPlot in v0.11 (which is the default, although though no windows will appear unless `show()` is called).

The error seems to happen during the plot(...), eg
```julia
begin
    using PyPlot
    plot()
end
```
segfaults for my install of Pluto 0.11 on Mac.

I upgraded matplotlib (v3.0.1 to v3.3.1) but it hasn't made a difference.

I tried to change the dependencies on my Mac by rebuilting PyPlot to use the default Python installation on my Mac instead of the miniconda install of PyCall, and I receive the same error though with a different version of `libQt5Gui.5.9.7.dylib`
```julia
julia> import Pluto; Pluto.run(1234)
Opening http://localhost:1234/ in your default browser... ~ have fun!

Press Ctrl+C in this terminal to stop Pluto

      From worker 2:	
      From worker 2:	signal (11): Segmentation fault: 11
      From worker 2:	in expression starting at none:1
      From worker 2:	_ZNK7QScreen6handleEv at /Users/samuel/opt/anaconda3/lib/libQt5Gui.5.9.7.dylib (unknown line)
      From worker 2:	Allocations: 15191208 (Pool: 15187095; Big: 4113); GC: 16
Worker 2 terminated.Distributed
.ProcessExitedException(2)
```

If other users aren't having this problem then I'm sure it's just a strange environment on my Mac, however I was surprised that `using PyPlot; plot()` runs fine on Pluto v0.10 and in the Julia REPL, but is segfaulting for me in Pluto v0.11 even if I keep the PyPlot/Python environments the same.

**Comment by j-fu** [comment]
Seems to be the same as #511

**Comment by adityakumar0** [comment]
I have the same problem. I'm running macOS Catalina 10.15.7. I went as far as deleting my entire Anaconda distribution thinking it was a problem with the base environment. I then deleted Julia 1.5.3. I reinstalled Julia 1.5.3 first (before installing Anaconda distribution) and then tried running a sample notebook that involved PyPlot. I got the same seg.fault error involving this PyQT GUI library. The method that sp94 proposed worked for me. Unfortunately, the method involving 'Agg' backend environment setting did not work for me.

My browser is up-to-date (Chrome). It's not a problem with Anaconda base environment interfering with the set-up. It may be 
that this error is caused by the recent Julia update, the recent macOS Catalina update, and maybe even the default python environment that gets shipped with macOS. 

Just to clarify, I double-checked whether I did a complete and clean uninstall of Anaconda distribution (including all libraries and python3).
